task_DEV_2/task_DEV_2/Program.cs

```diff
@@ -4,46 +4,57 @@ namespace task_DEV_2
 {
     class Program
     {
-        static void Main(string[] args)
+        static void InputArrayElements(int[] inputNumbers, int arrayLength)
         {
-            Console.WriteLine("Enter the size of the array :");
-            int arrayLength = Convert.ToInt32(Console.ReadLine());
-
-            if (arrayLength <= 20)
+            for (int i = 0; i < arrayLength; i++)
             {
-                int[] inputNumbers = new int[arrayLength];
+                Console.WriteLine($"Enter an array element № {i}");
+                inputNumbers[i] = Convert.ToInt32(Console.ReadLine());
+            }
+        }
+        static void ChangeMinMaxElements(int[] inputNumbers, int arrayLength)
```

Comment:
Please add empty line between the methods.

Reply:
done